tests/Doctrine/Tests/Common/Reflection/RuntimePublicReflectionPropertyTest.php

```diff
@@ -78,7 +78,7 @@ class RuntimePublicReflectionPropertyTestProxyMock implements Proxy
     private $initializer     = null;
 
     /**
-     * @var \Closure|null
+     * @var \Closure|bool
```

Comment:
Nope. There's an `inheritDoc` and phpDoc in Proxy::__isInitialized() is correct.